Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/transfers/cookiePolicy.cy.ts

```diff
@@ -14,13 +14,15 @@ describe('Cookie Policy', () => {
     })
 
     it('Should consent to cookies from cookie header button', () => {
+      cy.wait(2000);
       cy.getCookie('.ManageAnAcademyTransfer.Consent')
-      .should('exist')
-      .should('have.property', 'value', 'True')
+        .should('exist')
+        .should('have.property', 'value', 'True')
     });
 
     it('Should hide the cookie banner when consent has been given', () => {
-      cy.get("[data-test='cookie-banner']").should('not.exist')
+      cy.get("#acceptCookieBanner").should('have.css', 'display', 'block');  // Ensure the new acceptance message is visible
```

Comment:
Better to use `.is(':visible')` here but not the end of the world.

Reply:
You are right I have just created a new PR with this change and pushed it, if you don't mind reviewing.

PR link; https://github.com/DFE-Digital/prepare-academy-conversions/pull/1274